--- liblangtag-0.6.5/configure.ac.orig	2023-11-24 12:26:58.000000000 +0100
+++ liblangtag-0.6.5/configure.ac	2023-11-25 17:50:51.233768270 +0100
@@ -29,17 +29,17 @@ AC_PROG_CC_STDC
 AX_IS_RELEASE([git-directory])
 AX_COMPILER_FLAGS()
 
-warn=()
+warn=""
 for w in $WARN_CFLAGS; do
 	case $w in
 	-Wdeclaration-after-statement)
 		;;
 	*)
-		warn+=($w)
+		warn="$warn $w"
 		;;
 	esac
 done
-WARN_CFLAGS="${warn[[@]]}"
+WARN_CFLAGS="$warn"
 
 LT_PREREQ([2.2])
 LT_INIT([disable-static win32-dll])
