Update following data/common/supplemental/supplementalMetadata.xml change
--- liblangtag-0.6.4/tests/check-tag.c.orig	2021-09-27 06:52:04.000000000 +0200
+++ liblangtag-0.6.4/tests/check-tag.c	2023-03-04 15:04:16.449713051 +0100
@@ -197,7 +197,7 @@ TDEF (lt_tag_canonicalize) {
 	fail_unless(lt_tag_parse(t1, "mo", NULL), "should be valid langtag.");
 	s = lt_tag_canonicalize(t1, NULL);
 	fail_unless(s != NULL, "Unable to be canonicalize.");
-	fail_unless(lt_strcmp0(s, "ro-MD") == 0, "Unexpected result to be canonicalized.");
+	fail_unless(lt_strcmp0(s, "ro") == 0, "Unexpected result to be canonicalized.");
 	free(s);
 
 	lt_tag_unref(t1);
